spyder/plugins/plots/tests/test_plugin.py

```diff
@@ -27,8 +27,12 @@ def plots_plugin(qapp, qtbot):
     plots.get_widget().setMinimumSize(700, 500)
     plots.get_widget().add_shellwidget(Mock())
     qtbot.addWidget(plots.get_widget())
-    qapp.setStyleSheet(str(APP_STYLESHEET))
-    plots.get_widget().show()
+    # TODO: Setting app style causes test run to segfault over CI
+    # should the style be set elsewhere?
+    # qapp.setStyleSheet(str(APP_STYLESHEET))
```

Comment:
```suggestion
    if not running_in_ci():
        qapp.setStyleSheet(str(APP_STYLESHEET))
```

- I didn't know doing this was causing segfaults on CIs but it's useful for local testing.
- Please add the missing import.